ner_v2/detectors/numeral/number/zh-TW/number_detection.py

```diff
@@ -46,6 +45,8 @@ def __init__(self, entity_name='number', unit_type=None):
             '+': '加'
         }
 
+        self.power_of_10 = {10 ** i for i in range(1, 17)}
```

Comment:
Please perform multiplication over power operation to achieve the same.

Reply:
changes are done